pkg/cli/admin/upgrade/channel/channel.go

```diff
@@ -99,7 +99,7 @@ func (o *Options) Run() error {
 	}
 
 	if len(cv.Status.Desired.Channels) > 0 {
-		found, known := false
+		found, known := false, false
```

Comment:
nit: can we squash fixups like this down into the commit they're fixing up?  Future devs wondering why the code is a particular way may `git blame ...` the lines, and they don't need to hear about things like "briefly while the PR was in-flight, there was a minor syntax typo".